ee/tabby-schema/src/schema/repository/mod.rs

```diff
@@ -40,7 +40,7 @@ pub enum RepositoryKind {
     GitConfig,
 }
 
-#[derive(Debug)]
+#[derive(Debug, Clone)]
```

Comment:
is clone actually necessary? Can it be avoided by passing reference? (e.g &Repository)